### PR TITLE
April release

### PR DESCRIPTION
**Pull request by rsachan8**
1. storedCredentialUsed changed from boolean to string.
2. New API for KMS (/kms/v2/keys-sym/verifi)

### CODE REVIEW

**Comment by gnongsie** [comment]
Release is approved

File: cybersource_rest_client.gemspec [hunk 1]
```diff
@@ -30,9 +30,9 @@ Gem::Specification.new do |s|
 
   s.add_runtime_dependency 'typhoeus', '~> 1.0', '>= 1.0.1'
   s.add_runtime_dependency 'json', '~> 2.1', '>= 2.1.0'
-  s.add_runtime_dependency 'activesupport', '~> 6.0', '>= 6.0.3.2'
+  s.add_runtime_dependency 'activesupport', '>= 6.0.3.2', '< 8.0'
   s.add_runtime_dependency 'interface','~> 1.0', '>= 1.0.4'
-  s.add_runtime_dependency 'jwt', '~> 2.1'
+  s.add_runtime_dependency 'jwt', '2.1'
```
Comment (TomA-R) [inline cybersource_rest_client.gemspec:35]: hey @gnongsie, is this intentional?

Comment (gnongsie) [reply cybersource_rest_client.gemspec:35]: Hi @TomA-R, yes this is intentional. Due to changes to the jwt library, we have to restrict to certain versions.

If you have a suggestion for a better restriction, kindly let me know and I will have the team look into it.

Comment (TomA-R) [reply cybersource_rest_client.gemspec:35]: Locking `jwt` to `2.1` means everyone who uses this gem is locked into that. That means no security updates, fixes, or enhancements (like the OpenSSL 3.0 work they've been doing).. Is there any way we can either loosen the requirement or just bump it to the latest version?

**Pull request merged**